### ISSUE TITLE
The upper left circle with tilt capacity is gone?

### ISSUE THREAD

**Issue by borderthinking** [barryhunter/classygmaps#92]
What I see now looks like the new google view, with the wonderful classic circle with 4 arrow-tilt capacity gone from the upper left corner, now replaced by the primitive 2-directional thing in the lower right corner. That's a loss.

**Comment by barryhunter** [comment]
Ugg, hadnt really spotted that. Yes, the new style lacks directional compass ring. It is mentioned here:
https://developers.google.com/maps/articles/v322-controls-diff
in the Rotate Control section, but it doesn't really make clear the difference. The conveniently omit, the old ring around the pan control in comparing the differences. 

Not a lot I can do really. But for the moment can still get the old style control
http://gokml.net/maps-azteca.php
which will only work for another year. So I guess if you can might as well try get used to the new controls.

**Comment by barryhunter** [comment]
btw, not really sure Google will consider this a 'bug' that they will fix, as they've made deliberate choices with this new styleing. But I have at least reported it to Google that they should update the documentation to be more honest about the loss
https://code.google.com/p/gmaps-api-issues/issues/detail?id=8652

**Comment by borderthinking** [comment]
I didn't call it a compass rose because I use the tilt-mechanism in street
view, where it lifts up or down rather than north or south to give
different views of facades, for example. Nice of you to put it back. One
can nevr get the same effect by spinning the red & white arrow-thing in the
lower left. It's a dead loss.

**Comment by jnhunter** [comment]
I'd really miss the continuous rotate and tilt facility - with this 'new' (actually very old, I think) thing you can only rotate in 90 degree chunks which is bad.  What's the intention of the http://gokml.net/maps-azteca.php link, is this being maintained in parallel?  I'm intending to use it as primary.

**Comment by barryhunter** [comment]
Ah right, are you perhaps talking about streetview? Not thought about that. Thinking about the 45d imagry, which is only at 90d incrment, there is no continuus rotation. 

For streetview, just 'drag' the world itself, can do continuous rotation directly. I never used the control to rotate/tilt before. 

Can use the azteca page, but bear in mind Google plan to turn it off. 

If there is interest in azteca controls, could merge it back to the main map (making it a toggle)

**Comment by borderthinking** [comment]
I brought this up originally because of streetview and I still long for the
other way. In my tech world the dragging requires two hands, which the
tilty thing doesn't. Are you calling that azteca?

**Comment by jnhunter** [comment]
Yes, this applies to streetview - I didn't realise you could continuously pan/tilt by dragging the view, that works fine for me - I think the azteca style controls are preferable but these 'new' (other) ones are functionally OK
